pkg/apis/approvaltask/v1alpha1/approvaltask_types.go

```diff
@@ -33,10 +32,34 @@ type ApprovalTask struct {
 
 	// Spec holds the desired state of the TaskGroup from the client
 	// +optional
-	Spec ApprovalTaskSpec `json:"spec"`
+	Spec   ApprovalTaskSpec   `json:"spec"`
+	Params []Param            `json:"params,omitempty"`
+	Status ApprovalTaskStatus `json:"status"`
 }
 
 type ApprovalTaskSpec struct {
+	Approvals         []Input `json:"approvals"`
```

Comment:
IMO, `Approvals` => `Approvers`
Approvers will be the name of the approver right? []string is enough right?